crates/dslab-mp/src/process.rs

```diff
@@ -1,14 +1,30 @@
+use dyn_clone::{clone_trait_object, DynClone};
+
 use crate::context::Context;
 use crate::message::Message;
 
-pub trait Process {
+pub trait ProcessState {}
```

Comment:
Надо потребовать Clone:

```suggestion
pub trait ProcessState: Clone {}
```

Reply:
Вроде больше не нужно, я убрал клонирование в PyProcess